### PR TITLE
UI (sidebranch) test reorg -- PKI

### PR DESCRIPTION
**Pull request by hashishaw**
There were a lot of changes specific to PKI. An overview:

- Update component selector files to be in `helpers/components`
- Update exports from selectors to be named and non-default
- Don't import and reexport spread selectors 
- Remove PKI-specific selectors where we can use the generic ones
- Use generic selectors whenever possible
- Move certificates to top-level `pki/pki-workflow-helpers` file

### CODE REVIEW

File: ui/tests/helpers/components/pki/pki-issuer-cross-sign.js [hunk 1]
```diff
@@ -3,21 +3,15 @@
...
  */
 
-import { SELECTORS as CONFIGURE } from './pki-configure-create';
-import { SELECTORS as DETAILS } from './pki-issuer-details';
-
-export const SELECTORS = {
+export const PKI_CROSS_SIGN = {
   objectListInput: (key, row = 0) => `[data-test-object-list-input="${key}-${row}"]`,
-  inputByName: (name) => `[data-test-input="${name}"]`,
+  inputByName: (name) => `[data-test-input="${name}"]`, // GENERAL.inputByAttr()
```
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/tests/helpers/components/pki/pki-issuer-cross-sign.js:8]: No, this is a note to replace all instances. Still WIP :)

File: ui/tests/integration/components/pki/page/pki-configure-create-test.js [hunk 2]
```diff
@@ -40,22 +41,22 @@ module('Integration | Component | page/pki-configure-create', function (hooks) {
...
 
-    await click(SELECTORS.optionByKey('generate-root'));
-    assert.dom(SELECTORS.optionByKey('generate-root')).isChecked('Selected item is checked');
+    await click(PKI_CONFIGURE_CREATE.optionByKey('generate-root'));
+    assert.dom(PKI_CONFIGURE_CREATE.optionByKey('generate-root')).isChecked();
 
-    await click(SELECTORS.generateRootCancel);
-    assert.ok(this.cancelSpy.calledOnce);
+    await click(GENERAL.cancelButton);
+    assert.ok(this.cancelSpy.calledOnce, 'cancel action is called');
```
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/tests/integration/components/pki/page/pki-configure-create-test.js:60]: What would we replace it with? We want to ensure that it was called

File: ui/tests/integration/utils/parse-pki-cert-test.js [hunk 3]
```diff
@@ -12,18 +12,20 @@ import { Certificate } from 'pkijs';
 import { addHours, fromUnixTime, isSameDay } from 'date-fns';
 import errorMessage from 'vault/utils/error-message';
 import { OTHER_OIDs, SAN_TYPES } from 'vault/utils/parse-pki-cert-oids';
-import {
+import { verifyCertificates } from 'vault/utils/parse-pki-cert';
+import { jsonToCertObject } from 'vault/utils/parse-pki-cert';
+import { verifySignature } from 'vault/utils/parse-pki-cert';
```
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/tests/integration/utils/parse-pki-cert-test.js:17]: Yes.... strange 🤔

File: ui/tests/integration/components/pki/page/pki-configure-create-test.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/tests/integration/components/pki/page/pki-configure-create-test.js:60]: Oh maybe `assert.true` 😄

**Review by noelledaley — APPROVED** [review]
yesssss i would like, share, and subscribe to these changes ✅ ✅ ✅

**Comment by hashishaw** [comment]
Unfortunately closing this one due to gnarly merge conflicts, but the work and updates are represented in #26275

**Pull request closed without merging**